src/components/overview/Overview.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import ProductInfo from './ProductInfo/ProductInfo.js';
+import ProductDescription from './ProductDescription/ProductDescription.js'
```

Comment:
So uh.... we took out images, but added description, which is also not in the PR